pkgs/tools/security/pcsc-safenet/default.nix

```diff
@@ -26,13 +35,13 @@ stdenv.mkDerivation rec {
   '';
 
   buildInputs = [
-    gtk2
-    openssl
+    gtk3
+    openssl_1_1
     pcsclite
   ];
 
   runtimeDependencies = [
-    openssl
+    openssl_1_1
```

Comment:
Hm, we want to kill OpenSSL 1.1, is there a big reason for why OpenSSL3 wouldn't work?

Reply:
Good questions. The docs imply it _should_ be supported, but in practice it very much does not. Getting a closed-source binary blob, unfortunately, it's difficult to dig further.

Reply:
Got it! Keep in mind that https://www.openssl.org/blog/blog/2023/03/28/1.1.1-EOL/ :).